tests/MonologCreator/Factory/HandlerTest.php

```diff
@@ -189,6 +180,9 @@ public function testCreateUdpFailNoPort()
             true
         );
 
+        $this->expectException(\MonologCreator\Exception::class);
+        $this->expectExceptionMessage('port configuration for udp handler is missing');
```

Comment:
In general, wouldn't it improve readability if we put these expectations at the very top of a test function, so that when you read the code you directly see what is expected to happen?

Reply:
hmm I'm not sure. I followed the pattern:
- initialze/set local variables
- set expectations
- calling the actual code